src/Helper.js

```diff
@@ -0,0 +1,10 @@
+export function getRandomUniqueNumbersArray(max) {
+    const randomNumArr = [];
+    while (randomNumArr.length < max) {
+        const random = Math.floor(Math.random() * 5);
```

Comment:
Seems like Math.floor(Math.random() * 5) will generate from 0 to 5. In case max = 10 we will not get 10 unique numbers in this range. Lets fix this range.